### PR TITLE
Clarify encoding used for text by the File class

### PR DESCRIPTION
**Pull request by shartte**

### CODE REVIEW

File: doc/classes/File.xml [hunk 1]
```diff
@@ -83,7 +83,7 @@
 			<return type="String">
 			</return>
 			<description>
-				Returns the whole file as a [String].
+				Returns the whole file as a [String]. Text is interpreted as being UTF-8 encoded.
```
Comment (YeldhamDev) [inline doc/classes/File.xml]: This should be put below the existent text, to keep it consistent with the rest of the changes.

**Comment by YeldhamDev** [comment]
Remember to squish those commits as well.

**Comment by shartte** [comment]
> 
> 
> Remember to squish those commits as well.

I used the web editor for this one, I don't think I can squish it there. Can't you squish it on merge in the UI?

**Comment by reduz** [comment]
this is right

**Comment by akien-mga** [comment]
Force pushed a squashed commit.

**Comment by akien-mga** [comment]
Thanks!

**Comment by akien-mga** [comment]
Cherry-picked for 3.1.1.

**Pull request merged**